akka-stream-tests/src/test/scala/akka/stream/scaladsl/GraphFlexiMergeSpec.scala

```diff
@@ -186,12 +185,11 @@ class GraphFlexiMergeSpec extends AkkaSpec {
           merge.out ~> o.inlet
       }.run()
 
-      val s = SubscriberProbe[String]
+      val s = TestSubscriber.probe[String]
       p.subscribe(s)
       val sub = s.expectSubscription()
       sub.request(10)
-      (s.probe.receiveN(6).map { case OnNext(elem) ⇒ elem }).toSet should be(
-        Set("a", "b", "c", "d", "e", "f"))
+      (for (_ ← 1 to 6) yield s.expectNext()).toSet should be(Set("a", "b", "c", "d", "e", "f"))
```

Comment:
make this a pattern, probe.expectNextN for example. Maybe it should also handle completion, truncating the Seq if completed early (this would make the "Await(x.grouped().runWith(Sink.head)) should be (y)" pattern go await, since it would be x.runWith(probe).expectNextN() should be(y) given that probe also extends Sink)